src/components/common/ui/comment/Comment.styles.ts

```diff
@@ -0,0 +1,33 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+import typography from '@/styles/theme/constants/typography';
+
+const wrapper: SxProps<Theme> = {
+  maxWidth: '740px',
+  height: '100%',
+  padding: '11px 16px 12px 16px',
+  backgroundColor: 'backgroundDark.200',
+  borderRadius: '4px',
+};
+
+const text: SxProps<Theme> = {
+  marginBottom: '2px',
+  textAlign: 'justify',
+  typography: {
+    mobile: typography.body1,
```

Comment:
Like in a `backgroundColor`, you can write just `body1`. Same for other typographies